### ISSUE TITLE
Didn't find op for builtin opcode 'RESIZE_BILINEAR' version '3'

### ISSUE THREAD

**Issue by vatsDivyank** [tensorflow/tensorflow#49207]
System Information:
TensorFlow 2.3
Model Architecture - MobileNet V1
Arduino_TensorFlow Lite 2.4.0 Alpha

I fined Tuned Mobilenet V1 for face recoginition and later quantized using full integer Quantization Technique and later converted it to .cc format to upload it to Arduino Nano 33 BLE Sense. I used all_ops_resolver function but when i ran the Model on Microcontroller, invoke failed() and error message which was shown on Serial Monitor was " Didn't find op for builtin opcode 'RESIZE_BILINEAR' version '3'. An older version of this builtin might be supported. Are you using an old TFLite binary with a newer model?". Can anyone suggest me, how i can work around this error message.
After looking through the Arduino_TensorFlow Lite 2.4.0 - APLHA , i saw that resize_bilinear() isnt available in all_ops_resolver library but it is there in the master version of TensorFlow Lite on Tensorflow Github repository.  
Can anyone suggest me, how i overcome error message.

Colab Link - https://github.com/vatsDivyank/Face-Recoginition-TinyML/blob/985e769a84ecdd5ec8890d2288d262bbfa7cc18c/User_not_User_FineTuned.ipynb

**Comment by bhack** [comment]
Yes this opcode is quite fresh. See https://github.com/tensorflow/tensorflow/pull/48617

**Comment by vatsDivyank** [comment]
I tried to generate the arduino Library using
./tensorflow/lite/micro/tools/ci_build/test_arduino.sh
but the generated library is still not completly function. Is there a stable version already available?

**Comment by sachinprasadhs** [comment]
All the TFLite micro related code has been migrated to different repository here https://github.com/tensorflow/tflite-micro, could you try again with the latest version and the specific path for the above comment can be found here https://github.com/tensorflow/tflite-micro/tree/main/tensorflow/lite/micro/tools/ci_build.
